simplified-ui-accounts/src/main/java/org/nypl/simplified/ui/accounts/AccountListAdapter.kt

```diff
@@ -41,83 +40,57 @@ class AccountListAdapter(
   override fun onBindViewHolder(
     holder: AccountViewHolder,
     position: Int
-  ) {
-    holder.bind(this.getItem(position))
-  }
+  ) = holder.bind(getItem(position), itemCount != 1)
 
   class AccountViewHolder(
-    val itemView: View,
+    val binding: AccountListItemBinding,
     private val imageLoader: ImageLoaderType,
     private val onItemClicked: (AccountType) -> Unit,
     private val onItemDeleteClicked: (AccountType) -> Unit
-  ) : RecyclerView.ViewHolder(itemView) {
-    private val accountIcon =
-      itemView.findViewById<ImageView>(R.id.accountIcon)
-    private val accountTitleView =
-      itemView.findViewById<TextView>(R.id.accountTitle)
-    private val accountCaptionView =
-      itemView.findViewById<TextView>(R.id.accountCaption)
-    private val popupMenuIcon =
-      itemView.findViewById<View>(R.id.popupMenuIcon)
+  ) : RecyclerView.ViewHolder(binding.root) {
+
+    fun bind(account: AccountType, hasOptionsMenu: Boolean) {
+      binding.account = account
+      binding.executePendingBindings()
 
-    private var accountItem: AccountType? = null
+      binding.root.setOnClickListener {
+        onItemClicked(account)
+      }
+
+      imageLoader.loader.cancelRequest(binding.accountIcon)
+      ImageAccountIcons.loadAccountLogoIntoView(
+        loader = imageLoader.loader,
+        account = account.provider.toDescription(),
+        defaultIcon = R.drawable.account_default,
+        iconView = binding.accountIcon
+      )
 
-    init {
-      this.itemView.setOnClickListener {
-        this.accountItem?.let { account ->
-          this.onItemClicked.invoke(account)
+      if (hasOptionsMenu) {
+        binding.popupMenuIcon.setOnClickListener {
+          showPopupMenu(account)
         }
+      } else {
+        binding.popupMenuIcon.visibility = View.GONE
       }
```

Comment:
_I_ would have disabled the menu or just the Delete button instead of hiding it. I know Mark suggested the opposite and that's not a big deal though.